src/Kafka/ManageKafkaPermissions/components/ConsumeTopicShortcut.test.tsx

```diff
@@ -14,28 +14,7 @@ describe("Consume Topic shortcut permissions", () => {
     const comp = render(<FormSubmitted />);
 
     await waitForI18n(comp);
-
-    expect(await comp.findByText("Resource")).toBeInTheDocument();
-    expect(await comp.findByText("Permission")).toBeInTheDocument();
-    expect(await comp.findByText("Consume from a topic")).toBeInTheDocument();
-    expect(await comp.findByText("Write")).toBeInTheDocument();
-    expect(await comp.findByText("Create")).toBeInTheDocument();
-    expect(await comp.findByText("Read")).toBeInTheDocument();
-    expect(await comp.findByText("Describe")).toBeInTheDocument();
-    expect(await comp.findByText("Topic")).toBeInTheDocument();
-    expect(await comp.findByText("Consumer group")).toBeInTheDocument();
-    const requiredValue = await comp.findAllByText("Required");
-    expect(requiredValue[0]).toBeInTheDocument();
-    expect(requiredValue[1]).toBeInTheDocument();
-    const allow = await comp.findAllByText("Allow");
-    expect(allow[0]).toBeInTheDocument();
-    expect(allow[1]).toBeInTheDocument();
-    const startsWith = await comp.findAllByText("Starts with");
-    expect(startsWith[0]).toBeInTheDocument();
-    expect(startsWith[1]).toBeInTheDocument();
-    const prefix = await comp.findAllByPlaceholderText("Enter prefix");
-    expect(prefix[0]).toBeInTheDocument();
-    expect(prefix[1]).toBeInTheDocument();
+    expect(comp.baseElement).toMatchSnapshot();
```

Comment:
We should refrain from using snapshots as much as possible. It's ok to use them for static components with no logic or user interaction, it's not really useful for more complex things.

Reply:
I've added some more tests, using the interactive example to cover as many cases as possible. The individual components used in the view already had tests written for them